scrapping/vinted-db-feeder/lib/get_categories.py

```diff
@@ -0,0 +1,114 @@
+import json
+
+class Category:
+    def __init__(self, id: int, title, code: str, parent_id:int, url,
+                 url_en, children_ids: list[int]):
+        self.id = id
+        self.title = title
+        self.code = code
+        self.parent_id = parent_id
+        self.url = url
+        self.url_en = url_en
+        self.children_ids = children_ids
+        self.children = []
+
+    def __str__(self):
+        return f"{self.id},'{self.title}','{self.code}',{self.parent_id},'{self.url}','{self.url_en}'"
+
+    @staticmethod
```

Comment:
why this if you have arg self ?

Reply:
NVM, it was the @staticmethod, fixed it in the last commit.